uiowa/uiowa
# [Registrar] Academic calculator styling.

Closes https://github.com/uiowa/uiowa/issues/7933

## To test

1. Sync the registrar site
2. go to https://registrar.uiowa.ddev.site/gpa-calculator
3. Mess with the calculator. ensure it still works!
4. Do a code review. Is there something that I goofed?
